diff --git a/conf.py b/conf.py
index 282128179..76233a19e 100644
--- a/conf.py
+++ b/conf.py
@@ -335,7 +335,7 @@ latex_additional_files = [
 
 locale_dirs = ['locale/']   # Where the PO files will be stored at
 gettext_compact = False     # optional.
-gettext_additional_targets = ['image', 'index', 'literal-block'] # allows images to be translatable
+gettext_additional_targets = ['index', 'literal-block'] # allows images to be translatable
 figure_language_filename = "{path}{language}/{basename}{ext}"
 
 
